WebDevStudios/wd_s
# Feature / Update Tailwind to 3.2.1

### DESCRIPTION

- Update tailwind to latest version to allow support of multiple configs (in conjunction with wds-acf-blocks plugin)

*Note*
`svg-spritemap-webpack-plugin` was no longer installing with previous git url. This has been updated to the latest version as well with this PR

### OTHER


### STEPS TO VERIFY

- run `npm i` to update package
- run `npm run build` and confirm no issues and check theme frontend 
- confirm SVG sprite works
